docs/book/src/getting-started.md

```diff
@@ -83,6 +83,8 @@ we will allow configuring the number of instances with the following:
 ```go
 type MemcachedSpec struct {
 	...
+	// +kubebuilder:validation:Minimum=0
```

Comment:
Is this a valid user choice? What does size `0` mean?

Reply:
Yes, that is. 
If I want scale down the instance for any reason I could set 0